x/hub-genesis/keeper/ibc_module.go

```diff
@@ -95,26 +109,53 @@ func (w IBCModule) mintAndTransfer(
 	// or commit anyway, so the packet will be cleared up.
 	// (Actually, since transfers may arrive out of order, we must include the
 	// denom metadata anyway).
-	memo, err := w.createMemo(ctx, a.Amount.Denom, n)
+	memo, err := w.createMemo(ctx, account.Amount.Denom)
 	if err != nil {
 		return errorsmod.Wrap(err, "create memo")
 	}
 
 	m := transfertypes.MsgTransfer{
 		SourcePort:       portID,
 		SourceChannel:    channelID,
-		Token:            a.Amount,
+		Token:            account.Amount,
 		Sender:           srcAddr,
-		Receiver:         a.GetAddress(),
+		Receiver:         account.GetAddress(),
 		TimeoutHeight:    clienttypes.Height{},
 		TimeoutTimestamp: uint64(ctx.BlockTime().Add(transferTimeout).UnixNano()),
 		Memo:             memo,
 	}
 
-	err = w.transfer(skipContext(ctx), &m)
+	res, err := w.transfer.Transfer(sdk.WrapSDKContext(allowSpecialMemoCtx(ctx)), &m)
 	if err != nil {
 		return errorsmod.Wrap(err, "transfer")
 	}
-
+	w.k.saveUnackedTransferSeqNum(ctx, res.Sequence)
 	return nil
 }
+
+func (w IBCModule) OnAcknowledgementPacket(
+	ctx sdk.Context,
+	packet channeltypes.Packet,
+	acknowledgement []byte,
+	relayer sdk.AccAddress,
+) error {
+	l := w.logger(ctx)
+	state := w.k.GetState(ctx)
+	if !state.OutboundTransfersEnabled && // still in genesis protocol
```

Comment:
why not left indent with standart err check for each `if` clause?

```suggestion
	if state.OutboundTransfersEnabled {
        return
    }
    if !state.IsCanonicalHubTransferChannel() {
        //err
    }
    ...

```

Reply:
yeah sorry
I've made it flatter now